template.distillery/PROJECT_NAME_icons.eliom

```diff
@@ -1,24 +1,37 @@
+(** This module defines an interface to create icons HTML element with
+ * predefined style/value. It is supposed "Font Awesome" icons are used by
+ * default (fa CSS class is added when using [icon classes]).
+ * See http://fontawesome.io/ for more information and for the complete list of
+ * CSS classes values.
+ *)
+
 [%%shared.start]
 
 module Make(A : module type of Eliom_content.Html.F) = struct
 
-  let icon classes ?(class_=[]) () =
-    A.i ~a:[A.a_class ("fa"::classes@class_)] []
+  (** [icon classes] create an icon HTML attribute with "fa" and [classes]
+   * as CSS classes.
+   * The optional parameter is at the end to be able to add other CSS classes
+   * with predefined icons.
+   *)
+  let icon classes
+      ?(a = ([] : Html_types.i_attrib Eliom_content.Html.attrib list)) () =
+    A.i ~a:(A.a_class ("fa" :: classes) :: a) []
```

Comment:
`<i>` has traditionally meant italics. It is news to me that it is now widely used for icons :). It is OK if we do the same, but can you clarify that it is an `<i>` tag in the documentation comment?